src/hook/context.ts

```diff
@@ -1,9 +1,9 @@
-import React from "react";
-import ToastContainer from "../toast-container";
+import React from 'react';
+import ToastContainer from '../toast-container';
 
 export type ToastType = Pick<
-  ToastContainer,
-  "show" | "update" | "hide" | "hideAll" | "isOpen"
+    ToastContainer,
+    'show' | 'update' | 'hide' | 'hideAll' | 'isOpen' | 'toggle'
```

Comment:
toggle sounds like its calling either show or hide, maybe call it toggleToastsEnabled or something more descriptive

Reply:
Yeah, changed to toggleToastVisibility and state to isVisible